packages/manager/src/features/CloudPulse/Utils/FilterBuilder.ts

```diff
@@ -266,9 +295,14 @@ export const checkIfWeNeedToDisableFilterByFilterKey = (
     if (filter) {
       return filter.configuration.dependency?.some((dependent) => {
         const dependentFilter = dependentFilters[dependent];
+        const optionalFilter = filters.find(
+          (filter) => filter.configuration.isOptional === true
```

Comment:
```suggestion
          (filter) => filter.configuration.isOptional 
```
no need of equality check

Reply:
ok, will avoid redundant code, putting this in my checklist.